packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/observability/TelemetryServiceTest.kt

```diff
@@ -53,6 +53,7 @@ internal class TelemetryServiceTest {
                 nonGenuineServerName = null,
                 serverOsFamily = null,
                 isLocalAtlas = false,
+                atlasHost = "mongodb+srv://example-cluster.mongodb.net",
```

Comment:
Atlas host should be just hostname without a prefix, e.g. `example-cluster.mongodb.net`

Reply:
It's mock data, but yeah, just to be consistent I'll remove the prefix.

Reply:
Yeah, this will just be helpful to others who reads the code. I usually use tests as a learning tool 📚